crates/biome_js_analyze/tests/specs/nursery/useParseIntRadix/invalid.js.snap.new

```diff
@@ -0,0 +1,189 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+assertion_line: 86
+expression: invalid.js
+---
+# Input
+```jsx
+var num = parseInt("071");
+
+var num = parseInt(someValue);
+
+var num = parseInt("071", "abc");
+
+var num = parseInt("071", "8");
+
+var num = parseInt("071", 37);
+
+var num = parseInt();
+
+var num = Number.parseInt("10");
+
+var num = parseInt("0xF", 16n);
+
+var num = parseInt("071", "abc", 3);
+
+```
+
+# Diagnostics
+```
+invalid.js:1:11 lint/nursery/useParseIntRadix  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Missing radix parameter
+  
+  > 1 │ var num = parseInt("071");
+      │           ^^^^^^^^^^^^^^^
+    2 │ 
+    3 │ var num = parseInt(someValue);
+  
+  i Add a non-fractional number between 2 and 36
+  
+  i Unsafe fix: Remove add a radix of 10
+  
+    1 │ var·num·=·parseInt("071",·10);
+      │                         ++++  
+
+```
```

Comment:
Make sure to follow the rule pillars for writing good diagnostics: https://github.com/biomejs/biome/blob/main/crates/biome_analyze/CONTRIBUTING.md#explain-a-rule-to-the-user